CONTRIBUTING.md

```diff
@@ -83,16 +83,52 @@ Check out the next section for details on how this plays out.
 
 Some Bevy Org members are also Triage Team members. These people can label and close issues and PRs but do not have merge rights or any special authority within the community. Existing Org members can vist the Bevy Engine Org page and navigate to the Triage Team to request membership. Once again, if you are interested don't hesitate to apply. We generally accept membership requests.
 
-We heavily limit who has merge rights within the org because this requires a large amount of trust when it comes to ethics, technical ability, and ability to enforce consistent project direction. Currently, only @cart can merge every class of change. @mockersf is allowed to merge small "uncontroversial" changes, provided these changes have at least two approvals. Same goes for @alice-i-cecile and doc related changes. If there is an emergency that needs a quick resolution and @cart is not around, both @mockersf and @alice-i-cecile are allowed to merge changes that resolve the emergency.
+Merge rights within the org are relatively centralized: this requires a large amount of trust when it comes to ethics, technical ability, and ability to enforce consistent project direction.
+
+The current structure is as follows:
+
+* @cart is our project lead, and has final say on controversial decisions
+* There is a small group of other maintainers (@alice-i-cecile, @mockersf and @superdump), who have merge rights but abide by the following rules:
+  * Trivial PRs can be merged without approvals
+  * Relatively uncontroversial PRs can be merged following approval from at least one other community member with appropriate expertise.
```

Comment:
I'd personally prefer if this said "at least two community approvals", I don't like that my vote has that much power even if it's for something uncontroversial. If it's something important then multiple people will approve it, otherwise it's probably not a priority to many people.

Reply:
```suggestion
  * Relatively uncontroversial PRs can be merged following approval from at least two other community members with appropriate expertise.
```